fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/spec/UpdateOperation.java

```diff
@@ -19,18 +22,35 @@
 	@Override
 	public void process(TestContext tc) throws FHIRPersistenceException {
 	    
-        final Resource resource = tc.getResource();
+        Resource resource = tc.getResource();
         final FHIRPersistenceContext context = tc.createPersistenceContext();
 
         final String logicalId = resource.getId();
-        
         logger.fine("Updating: " + logicalId);
+        
+        // For 1869, we need to update the resource meta before we call the persistence layer
+        Instant lastUpdated = Instant.now(ZoneOffset.UTC);
+        final int newVersionNumber = Integer.parseInt(resource.getMeta().getVersionId().getValue()) + 1;
+        resource = copyAndSetResourceMetaFields(resource, resource.getId(), newVersionNumber, lastUpdated);
 
-        Resource newResource = tc.getPersistence().update(context, logicalId, resource).getResource();
-        check(tc, resource, newResource, this.getClass().getSimpleName());
+        Resource newResource = tc.getPersistence().update(context, logicalId, newVersionNumber, resource).getResource();
+        check(tc, tc.getResource(), newResource, this.getClass().getSimpleName());
         
         // Update the context with the modified resource
         tc.setResource(newResource);
 	}
 
-}
+    /**
+     * Creates and returns a copy of the passed resource with the {@code Resource.id}
+     * {@code Resource.meta.versionId}, and {@code Resource.meta.lastUpdated} elements replaced.
+     *
+     * @param resource
+     * @param logicalId
+     * @param newVersionNumber
+     * @param lastUpdated
+     * @return the updated resource
+     */
+    private <T extends Resource> T copyAndSetResourceMetaFields(T resource, String logicalId, int newVersionNumber, com.ibm.fhir.model.type.Instant lastUpdated) {
+        return FHIRPersistenceUtil.copyAndSetResourceMetaFields(resource, logicalId, newVersionNumber, lastUpdated);
+    }
```

Comment:
Do we need a method for this? Can't FHIRPersistenceUtil.copyAndSetResourceMetaFields() just be called directly?